topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md

```diff
@@ -0,0 +1,424 @@
+---
+layout: tutorial_hands_on
+
+title: SARS-CoV-2 Viral Sample Alignment and Variant Visualization
+level: Introductory
+zenodo_link: ''
+questions:
+- How do I check my data quality?
+- How do I put together my dataset of fragmented sequences into a full sequence? 
+- How do I visually explore the variants in my SARS-CoV-2 sample?
+objectives:
+- Gather and evaluate experimental evidence, including qualitative and quantitative data
+- Generate and interpret graphs displaying experimental results
+- Critique large data sets and use bioinformatics to assess genetics data
+- Tap into the interdisciplinary nature of science
+time_estimation: "1H"
+key_points:
+- Use Galaxy on the AnVIL cloud computing resource to check data, perform an alignment, and visualize the results.
+contributions:
+    authorship:
+    - nakucher
+    - avahoffman
+    - robertmeller
+    editing:
+    - katherinecox
+    - cutsort
+    - nakucher
+    infrastructure:
+    - cansavvy
+    - avahoffman
+    - katherinecox
+    - ehumph
+    - cutsort
```

Comment:
if you have funding source you'd like to call out, you're welcome to annotate that here as well!

Reply:
added, thanks!